### ISSUE TITLE
Is this toolkit applicable to OCP3.10?

### ISSUE THREAD

**Issue by aelicieguo** [redhat-cop/openshift-toolkit#56]
As most OCP components are containerized, such as kube-system, openshift-node and openshift-sdn, is the basline networkpolicy applicable still?

**Comment by etsauer** [comment]
@aelicieguo I don't think we would recommend blanket applying the baseline policies against pre-existing namespaces, but instead applying it to all _new_ projects (i.e. application projects), possibly via a default project template for the cluster.

**Comment by garethahealy** [comment]
@aelicieguo ; has your question been answered? can this be closed?
